lib/Dialect/FIRRTL/LowerToRTL.cpp

```diff
@@ -163,6 +163,57 @@ static LogicalResult lower(firrtl::CatPrimOp op, ArrayRef<Value> operands,
   return success();
 }
 
+//===----------------------------------------------------------------------===//
+// Variadic Bitwise Operations
+//===----------------------------------------------------------------------===//
+
+#include <iostream> 
```

Comment:
Random driveby comment (please lemme know when you're ready for review, I realize this is a work in progress):

Please don't use <iostream> in CIRCT, it has several problems:
https://llvm.org/docs/CodingStandards.html#include-iostream-is-forbidden

Reply:
Yes, iostream was never going to be pushed; it's an artifact from the debugging pair programming, but thanks for the reminder to remove it.